d3a_api_client/rest_device.py

```diff
@@ -15,20 +18,28 @@
 
 class RestDeviceClient(APIClientInterface, RestCommunicationMixin):
 
-    def __init__(self, simulation_id, device_id,
+    def __init__(self, device_id=None, simulation_id=None,
                  domain_name=domain_name_from_env,
                  websockets_domain_name=websocket_domain_name_from_env,
                  autoregister=False, start_websocket=True,
                  sim_api_domain_name=None):
-        self.simulation_id = simulation_id
+        if os.environ['JSON_FILE_PATH'] is not None:
+            with open(os.environ['JSON_FILE_PATH']) as json_file:
+                simulation_info = json.load(json_file)
+                self.simulation_id = simulation_info['uuid']
+                self.domain_name = simulation_info['domain_name']
+                self.websockets_domain_name = simulation_info['web_socket_domain_name']
+        else:
+            self.simulation_id = simulation_id
+            self.domain_name = domain_name
+            self.websockets_domain_name = websockets_domain_name
+
```

Comment:
If possible can you move this logic into its own private function to keep the __init__ clean